jax/_src/scipy/stats/bernoulli.py

```diff
@@ -12,19 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-import scipy.stats as osp_stats
-
 from jax import lax
 import jax.numpy as jnp
 from jax._src.lax.lax import _const as _lax_const
-from jax._src.numpy.util import implements, promote_args_inexact
+from jax._src.numpy.util import promote_args_inexact
 from jax._src.typing import Array, ArrayLike
 from jax.scipy.special import xlogy, xlog1py
 
 
-@implements(osp_stats.bernoulli.logpmf, update_doc=False)
 def logpmf(k: ArrayLike, p: ArrayLike, loc: ArrayLike = 0) -> Array:
+  """Bernoulli log probability mass function.
+
+  JAX implementation of :func:`scipy.stats.bernoulli` ``logpmf``
```

Comment:
I assume we can't actually cross reference the function directly because it doesn't have a top-level header in scipy?

Reply:
Linking https://en.wikipedia.org/wiki/Probability_mass_function might be helpful. In general consider linking, say, the Wikipedia definitions of each stats term.

Reply:
> In general consider linking, say, the Wikipedia definitions of each stats term.

In the rendered version, each one links to the scipy docs, which contain more information. Do you think that's sufficient, or should we repeat that information here?

Reply:
Alright, I added a mathematical description inline for every function.